card/mod/machines/lib/javascript/decko/card_menu.js.coffee

```diff
@@ -0,0 +1,15 @@
+decko.isTouchDevice = ->
```

Comment:
no change, moved here from a decko.js.coffee and a separated script card  "script: card menu" that I removed